Source/items.cpp

```diff
@@ -2901,15 +2874,15 @@ void CreatePlrItems(int playerId)
 		break;
 	}
 
-	auto &startingGold = player.InvList[player._pNumInv];
+	GenerateNewSeed(GoldTemplate);
+	GoldTemplate._ivalue = 100;
+	GoldTemplate._iCurs = ICURS_GOLD_SMALL;
+
+	player.InvList[player._pNumInv] = GoldTemplate;

```

Comment:
This is modifying a global object used in loads of other places, if you're going to copy based on the template would be better to make the modifications to the `InvList` instance.